src/scss/includes/panels/poi_panel.scss

```diff
@@ -313,75 +289,36 @@ $HEADER_SIZE: 40px;
 }
 
 .poi_panel__actions {
-  width: 100%;
-  position: relative;
-  margin-bottom: 10px;
-  overflow: auto;
+  display: flex;
```

Comment:
Without this `margin-bottom`, the "osm_contribute" div is packed below the buttons (when no other block is present)
![image](https://user-images.githubusercontent.com/4726554/63034078-f5496a80-beb8-11e9-84bf-3dbf442023dc.png)

Maybe add a margin-top on `.osm_contribute` ?

Reply:
Thanks, I was a bit too quick to remove rules. I re-introduced a `margin-bottom`, as it's easier to manage only bottom margins when stacking components. It looks better indeed :)
![Capture d’écran de 2019-08-14 17-52-58](https://user-images.githubusercontent.com/243653/63035940-5e7ead00-bebc-11e9-8657-192f67c627de.png)